docs/src/architecture/assets.md

```diff
@@ -0,0 +1,53 @@
+# Assets
+The Miden Rollup aims to provide a reliable protocol and software for users to exchange assets. In Miden, users can create and trade arbitrary fungible and non-fungible assets. 
+
+We differentiate between native and non-native assets in Miden. Native assets follow the Miden asset model. Non-native assets are all other data structures of value that can be exchanged.   
+
+Recording of native assets in Polygon Miden suffices four goals:
+
+* Asset exchange should be parallelizable 
+* Asset ownership should be private 
+* Asset usage should be indeed censorship resistant
+* Fees can be paid using any asset
+
+All native assets in Miden are stored directly in accounts, like Ether in Ethereum. Miden does not track ownership of assets using global hashmaps, e.g., ERC20 contracts. Storage of assets locally in accounts provides privacy and the ability for client-side proofs. That is because ownership changes always only involve one account and not the change of a global hashmap. Thus, they can happen in parallel. Additionally, asset exchange is censorship resistant at this level because there is no global contract the transfer must pass through. Finally, users can pay fees in any asset. 
```

Comment:
That is because ownership changes always only involve one account and not the change of a global hashmap. -> This is because asset transfers involve a single account and do not require changes to global state outside of the account context.

Reply:
Very well. This needs to change in a separate PR. This PR is only about notes, it just didn't rebase properly. Will change in a separate PR.